src/parameterizations/lateral/MOM_hor_visc.F90

```diff
@@ -1799,12 +1806,44 @@ subroutine horizontal_viscosity(u, v, h, diffu, diffv, MEKE, VarMix, G, GV, US,
                  + str_xy(I,J-1) *                                &
                    ((u(I,j,k)-u(I,j-1,k))*G%IdyBu(I,J-1)          &
                   + (v(i+1,J-1,k)-v(i,J-1,k))*G%IdxBu(I,J-1)) ) ) )
-    enddo ; enddo ; endif
+        enddo ; enddo 
+      else ; do j=js,je ; do i=is,ie
+         FrictWork(i,j,k) = GV%H_to_RZ * G%IareaT(i,j) * ( &
+                (str_xx(i,j)*CS%dy2h(i,j) * ( &
+                              uh(I,j,k)*G%dxCu(I,j)/(h_u(I,j)+h_neglect)*G%IdyCu(I,j)*G%IareaCu(I,j) &
```

Comment:
I can never remember whether Fortran is left-associative or right-associative, and I should not have to.  Please rearrange or add parentheses to this and the other new expressions to make it clear to the reader which terms are in the numerator and the denominator, perhaps by rewriting `uh(I,j,k)*G%dxCu(I,j)/(h_u(I,j)+h_neglect)*G%IdyCu(I,j)*G%IareaCu(I,j)` as `(uh(I,j,k)*G%dxCu(I,j)/(h_u(I,j)+h_neglect)) * G%IdyCu(I,j) * G%IareaCu(I,j)` and similarly for the other expressions.

Also, we are about to add new sets of parentheses so that the code will pass rotational consistency testing with fused-multiply adds. To do this, we are changing many of the expressions that look like `A*B + C*D` into `(A*B) + (C*D)`, wherever the two terms would be swapped upon rotation by 90 or 180 degrees.  This applies here.  Please add parentheses around the new terms that are being added to or subtracted from each other.  (If you need more details on what I am asking for here, @Wendazhang33, please come see me.)

Reply:
Thanks for your suggestions. I have made the rearrangements and added parentheses. Hope this version will suffice. Let me know if there are any additional issues.